new/LIT7048Qene.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7048Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ከልአኪ፡ አሮን፡ በበትረ፡ እዴሁ፡ ጥልልት።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Kalʾa-ki ʾAron ba-batra ʾǝde-hu ṭǝllǝlt (Qǝne)</title>
```

Comment:
We can indicate: Qǝne, possibly of etana mogar type. 
Or: of undefined type. 
In the abstract we can comment that the type is possibly etana mogar.
This has 11 verses, but a different rhyme of the first two I cannot explain. Perhaps 
 it follows a special type or (regional) subtype, but I do not see it for the moment. By the way, as a reference I use Guidi's "Qene, or inni abissini", 1901, with a list of types  and examples (I will circulate a scan). There are some other survey of qene types, Guidi is good and precise. We can gradually collect a bibliography pn qene, this will not be too much (but there is an unknown number of Ethiopian editions).